src/icons/index.ts

```diff
@@ -1,6 +1,29 @@
 export { default as Add } from './Add'
+export { default as ArrowDropDown } from './ArrowDropDown'
```

Comment:
Could you update the demo as well to have all the icons? Maybe we could make it automatic by iterating over the module keys?

Reply:
Looks like you handled this one already.